cargo-fix/src/main.rs

```diff
@@ -163,7 +129,7 @@ fn rustfix_crate(rustc: &Path, filename: &str) -> Result<HashMap<String, String>
     //
     // TODO: this should be configurable by the CLI to sometimes proceed to
     // attempt to fix broken code.
-    if !output.status.success() {
+    if !output.status.success() && env::var_os("__CARGO_FIX_BROKEN_CODE").is_none() {
```

Comment:
I fear that we'll be using a truckload of env vars soon (but it's not painful enough yet to fix)